### DISCUSSION TITLE
[old] openvk discussions

### DISCUSSION THREAD

**Discussion by veselcraft** [OpenVK/openvk#11] [General]
This is a place where you can share ideas or ask us few questions.
Russian and English languages are allowed

Это место, где вы можете поделиться идеями или задам нам несколько вопросов
Для дискуссии разрешён Русский и Английский язык

**Comment by ghost** [comment]
Сделайте микроблог.

**Comment by ThatMaidGuy** [comment]
Беседы ебаните и уведомления - и будет заебись!

**Comment by WerySkok** [comment]
Open GitHub issues because other solutions are unusable.

**Reply by ghost** [reply]
Done.

**Comment by veselcraft** [comment]
Я короче понял, что тут юзвери сами должны создавать треды.. тыкайте по вкладке, есчо
